libs/langchain-qianfan/src/chat_models.ts

```diff
@@ -0,0 +1,546 @@
+import {
```

Comment:
Does this replace `BaiduWenxin`? Can we mark that as deprecated in `@langchain/community`?

Like this:

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-community/src/vectorstores/pinecone.ts#L37

Reply:
langchain-baidu-qianfan will replace BaiduWenxin. I marked this replacement in @langchain/community - BaiduWenxin